docs/mkdocs/docs/development/quick-start.md

```
@@ -0,0 +1,205 @@
+
+Depending on files changed in PR you might need to get required checks to pass first and get reviews from owners or
+maintainers, following are related GH docs:
+
+- [About Status Checks]
+- [Required reviews]
+
+[Quick start: Open a Git repository or GitHub PR in an isolated container volume]:
+  https://code.visualstudio.com/docs/devcontainers/containers#_quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume
+[Quick start: Open an existing folder in a container]:
+  https://code.visualstudio.com/docs/devcontainers/containers#_quick-start-open-an-existing-folder-in-a-container
+[named volume]: https://docs.docker.com/engine/storage/volumes/
```

Markdownlint (0.37.0)

Unused link or image reference definition: "named volume"
Link and image reference definitions should be needed

(MD053, link-image-reference-definitions)